Description: Remove shebang from bash completion file
 Discussed upstream, rejected as most code editors rely on the shebang
 to correctly indent and highlight the file.
Forwarded: https://github.com/docker/cli/pull/2853
Author: Arnaud Rebillout <elboulangero@gmail.com>
Last-Update: 2020-11-27

Index: docker/cli/contrib/completion/bash/docker
===================================================================
--- docker.orig/cli/contrib/completion/bash/docker
+++ docker/cli/contrib/completion/bash/docker
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+# bash completion for docker(1)                             -*- shell-script -*-
 # shellcheck disable=SC2016,SC2119,SC2155,SC2206,SC2207,SC2254
 #
 # Shellcheck ignore list:
@@ -5673,3 +5673,5 @@ eval "$__docker_previous_extglob_setting
 unset __docker_previous_extglob_setting
 
 complete -F _docker docker docker.exe dockerd dockerd.exe
+
+# ex: filetype=sh
